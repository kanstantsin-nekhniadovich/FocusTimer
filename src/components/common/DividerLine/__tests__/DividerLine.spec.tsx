import React from 'react';
import { render } from '@testing-library/react-native';

import { DividerLine } from '..';

describe('<DividerLine />', () => {
  it('show match snapshot', () => {
    const tree = render(<DividerLine />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
