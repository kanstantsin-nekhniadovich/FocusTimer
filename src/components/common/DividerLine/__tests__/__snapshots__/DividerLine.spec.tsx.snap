// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DividerLine /> show match snapshot 1`] = `
<View
  style={
    Object {
      "height": 16,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#9e9e9e",
        "height": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fbfbfb",
          "height": 16,
          "width": 58,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#9e9e9e",
            "textTransform": "uppercase",
          }
        }
      >
        or
      </Text>
    </View>
  </View>
</View>
`;
